components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationMgtValidator.java

```diff
@@ -357,20 +357,22 @@ private void validateClaimsConfigs(List<String> validationMsg, ClaimConfig claim
 
         ApplicationManagementService applicationMgtService = ApplicationManagementService.getInstance();
         String[] allLocalClaimUris = applicationMgtService.getAllLocalClaimUris(tenantDomain);
+        ArrayList<String> allRemoteClaimUris = new ArrayList<String>();
```

Comment:
shall we rename the variable to remoteClaimUris?

also no need to define type when initializing array list. we can directly call new ArrayList<>();

Reply:
addressed in 5f538eb3b5513b3978a784dd8a0df158de48ae34